src/main/java/global/IFrame.java

```diff
@@ -6,4 +6,6 @@ public interface IFrame {
     public void refresh();
     public void back();
     public void home();
+
+    void dispose();
```

Comment:
casting as JFrame will allow for disposal of the screen, however this may be cleaner